fedimint-core/Cargo.toml

```diff
@@ -43,6 +43,7 @@ rand = "0.8.5"
 miniscript = { version = "7.0.0", git = "https://github.com/rust-bitcoin/rust-miniscript/", rev = "2f1535e470c75fad85dbad8633986aae36a89a92", features = [ "compiler", "serde" ] }
 secp256k1-zkp = { version = "0.7.0", features = [ "use-serde", "bitcoin_hashes", "global-context" ] }
 macro_rules_attribute = "0.1.3"
+bitvec = "1.0.1"
```

Comment:
@dpc we're not concerned about this anymore? (https://github.com/fedimint/fedimint/issues/796)

Reply:
The author is reputable and did some push to sorting outstanding issues some time after pointed out that they are lingering. 

The reality is that most of these crates have unpaid maintainers that are spread thin.

If a dependency can be avoided (especially non-trivial), it's better one way or another though.